dotCMS/src/main/java/com/dotcms/graphql/datafetcher/CountDataFetcher.java

```diff
@@ -0,0 +1,27 @@
+package com.dotcms.graphql.datafetcher;
+
+import com.dotcms.graphql.DotGraphQLContext;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.contentlet.transform.FolderToMapTransformer;
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.model.User;
+
+import java.util.List;
+import java.util.Map;
+
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+
+public class CountDataFetcher implements DataFetcher<List<Map<String, Object>>> {
+    @Override
+    public List<Map<String, Object>> get(final DataFetchingEnvironment environment) throws Exception {
+        try {
+            return ((DotGraphQLContext) environment.getContext()).getFieldCountMaps();
+        } catch (Exception e) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5963862)